.rubocop.yml

```diff
@@ -1,5 +1,11 @@
 require:
-- 'rubocop-md'
+  # add after moving docs to another tool
```

Comment:
We have a problem with `semantic_blocks` cop along with `rubocop-md`. It crashes because of our `md` files contain headers like
```
---
id: active_record
title: Active Record
---
```
I don't think we need to waste time to fix it right now, because we want to move from  `Docusaurus` to another Docs tool and it will be fixed automatically (I hope)